### PR TITLE
qml: settings (old design)

### PR DESCRIPTION
**Pull request by dvajdual**

### CODE REVIEW

File: GUI/QtQuick/qml/Settings/SettingsGeneral.qml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    }
+
+    CustomCheckBox {
+        id: checkbox_advanced_tx
+
+        Layout.alignment: Qt.AlignVCenter | Qt.AlignLeft
+        Layout.topMargin: 24
+        Layout.leftMargin: 24
+
+        text: "Advanced TX dialog by default"
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsGeneral.qml]: missing `qsTr`

File: GUI/QtQuick/qml/Settings/SettingsGeneral.qml [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        id: save_but
+
+        Layout.bottomMargin: 40
+        Layout.alignment: Qt.AlignBottom | Qt.AlignHCenter
+
+        enabled: log_file.input_text.length && messages_log_file.input_text.length
+
+        width: 532
+
+        text: qsTr("Save")
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsGeneral.qml]: We don't have "Save" button in mockup. It's only for adding a new row into the Armory servers model (to be added later). Each setting should be saved by assigning to a property in corresponding control.

File: GUI/QtQuick/qml/Settings/SettingsMenu.qml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        Layout.alignment: Qt.AlignCenter
+        Layout.topMargin: 24
+
+        //properties
+        isButton: true
+
+        //aliases
+        icon_source: "qrc:/images/general.png"
+        icon_add_source: "qrc:/images/arrow.png"
+        tytle_text: "General"
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsMenu.qml]: No translation. Should be `title_text`.

File: GUI/QtQuick/qml/Settings/SettingsMenu.qml [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        Layout.alignment: Qt.AlignCenter
+        Layout.topMargin: 10
+
+        //properties
+        isButton: true
+
+        //aliases
+        icon_source: "qrc:/images/network.png"
+        icon_add_source: "qrc:/images/arrow.png"
+        tytle_text: "Network"
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsMenu.qml]: No translation. Should be `title_text`.

File: GUI/QtQuick/qml/Settings/SettingsMenu.qml [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+        Layout.alignment: Qt.AlignCenter
+        Layout.topMargin: 10
+
+        //properties
+        isButton: true
+
+        //aliases
+        icon_source: "qrc:/images/about.png"
+        icon_add_source: "qrc:/images/arrow.png"
+        tytle_text: "About"
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsMenu.qml]: Same as 2 occurences above.

File: GUI/QtQuick/qml/Settings/SettingsNetwork.qml [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        id: save_but
+
+        Layout.bottomMargin: 40
+        Layout.alignment: Qt.AlignBottom | Qt.AlignHCenter
+
+        enabled: host.input_text.length && port.input_text.length
+
+        width: 532
+
+        text: qsTr("Save")
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsNetwork.qml]: Same - no "Save" button is needed.

File: GUI/QtQuick/qml/StyledControls/CustomListItem.qml [hunk 7]
```diff
@@ -0,0 +1,103 @@
+/*
+
+***********************************************************************************
+* Copyright (C) 2018 - 2022, BlockSettle AB
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/StyledControls/CustomListItem.qml]: Copyright year reminder (it's 2023 now).

File: GUI/QtQuick/qml/Settings/SettingsGeneral.qml (hunk 2 above)
Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/SettingsGeneral.qml]: We have mockup with other functionaluty))) (I mean figma) But ok I will remove Save button

Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/SettingsGeneral.qml]: If we will change 1) port of armory (immediantly set data in c++) then 2) host of armory (immediantly set data in c++) . Between 1) and 2) step we will have wrong armory configuration - can be it problem or not?

File: GUI/QtQuick/qml/Settings/SettingsMenu.qml (hunk 3 above)
Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/SettingsMenu.qml]: Thanks

File: GUI/QtQuick/qml/Settings/SettingsGeneral.qml (hunk 2 above)
Comment (sergey-chernikov) [reply GUI/QtQuick/qml/Settings/SettingsGeneral.qml]: No, it won't harm now, program restart will be required for this setting to take effect (together with testnet/environment change).

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**